src/Microsoft.ServiceFabric.Actors/Remoting/ActorRemotingProviderAttribute.cs

```diff
@@ -61,12 +61,12 @@ public abstract class ActorRemotingProviderAttribute : Attribute
         /// </summary>
         protected ActorRemotingProviderAttribute()

```

Comment:
It looks like we will need to throw the InvalidOperationException from the ActorRemotingProviderAttribute.GetProvider as well, correct?